index.php

```diff
@@ -1,4 +1,12 @@
-<?php require_once __DIR__ . '/vendor/autoload.php'; ?>
+<?php require_once __DIR__ . '/vendor/autoload.php';
+use Store\Category as Category;
+use Store\DBConnect as DBConnect;
+use Store\Store as Store;
+$dBConnect = DBConnect::connectToDB();
+$store = new Store($dBConnect);
```

Comment:
could we introduce a static function :: to link new Store with getCategories method?

Reply:
done